spec/support/capybara_poltergeist.rb

```diff
@@ -0,0 +1,13 @@
+RSpec.configure do |config|
+  Capybara.default_max_wait_time = 60 # Seconds to wait before timeout error. Default is 2
+
+  Capybara.server_port = 52662
+
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, inspector: true, debug: false)
+  end
+
+  Capybara.default_driver = :poltergeist
```

Comment:
I don't think this is a good idea. You should use Rack test for normal specs without javascript as its much faster and still reliable.